biopet-extensions/src/main/scala/nl/lumc/sasc/biopet/extensions/gatk/MuTect2.scala

```diff
@@ -0,0 +1,233 @@
+package nl.lumc.sasc.biopet.extensions.gatk
+
+import java.io.File
+
+import nl.lumc.sasc.biopet.core.ScatterGatherableFunction
+import nl.lumc.sasc.biopet.utils.config.Configurable
+import org.broadinstitute.gatk.utils.commandline.{Argument, Gather, Input, Output}
+
+class MuTect2(val parent: Configurable) extends CommandLineGATK with ScatterGatherableFunction {
+  scatterClass = classOf[LocusScatterFunction]
+  setupScatterFunction = { case scatter: GATKScatterFunction => scatter.includeUnmapped = false }
+
+  def analysis_type: String = "MuTect2"
+
+  /** Bam file for the tumor sample. */
+  @Input(fullName = "tumor_bam", required = true)
+  var tumorSampleBam: File = _
```

Comment:
I have different opinion about this. To my mind only the wrapper should know what is the exact syntax for calling GATK's tool, it's not good to spread this elsewhere, that the caller of the wrapper should also know this. With nl.lumc.sasc.biopet.extensions.gatk.ContEst it's the same, why should the caller know that the tumor sample should have the tag 'eval' and that the normal sample has the tag 'genotype'. But indeed those vars could be removed and the input_file var from the superclass could be used instead, but that without requiring that the caller would have to use TaggedFile-s. Are you ok with this solution?